src/main/java/com/juniorstart/juniorstart/util/AppConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.juniorstart.juniorstart.util;
+
+public interface AppConstants {
```

Comment:
Przeniósłbym te stale do serwisu po prostu, ewentualnie do yml. Ale chyba do serwisu

Reply:
@Ejden  a to przypadkiem nie było ostatnio konsultowane, a wynikiem było właśnie pozostawienie tego gdzie aktualnie jest ?

Reply:
Nie pamiętam tej rozmowy :P Jutro mogę o tym porozmawiać z Grzesiem bo dzisiaj nie ma zbytnio czasu